visualization/app/codeCharta/codeCharta.component.scss

```diff
@@ -1,6 +1,5 @@
 @import "../mixins";
 
-code-charta-component {
```

Comment:
By removing this the sidebar doesn't adapt the with of 600px anymore...

Reply:
I will fix this.